testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -296,8 +296,8 @@ def test_rmsd_mismatched_weights_raises_ValueError(self, universe):
             RMSD = MDAnalysis.analysis.rms.RMSD(
                 universe, weights=universe.atoms.masses[:-1])
 
-    def test_rmsd_misuse_weights_for_groupselection_raises_TypeError(self, universe):
-        with pytest.raises(TypeError):
+    def test_rmsd_misuse_weights_for_groupselection_raises_ValueError(self, universe):
+        with pytest.raises(ValueError):
```

Comment:
This is the relevant test for the TypeError above.

Reply:
Corrected that.